CMakeLists.txt

```diff
@@ -130,7 +130,9 @@ if(BUILD_DEPENDENCIES)
   )
   build_dependency(srtp ${BUILD_ARGS})
 
-  build_dependency(usrsctp)
+  set(BUILD_ARGS
+      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
```

Comment:
Thanks @chertov . Can we set a default for this?

Reply:
@disa6302 ok, i set a default value to RELEASE in https://github.com/chertov/amazon-kinesis-video-streams-webrtc-sdk-c/blob/3850df646ec10e4de305c8d6b0a7f2e0fe8b1e08/CMakeLists.txt#L36